surveys/2021-annual-survey/questions.md

```diff
@@ -219,70 +219,213 @@ TODO ask about blockers for participation?
+- My company has seriously discussed but not experimented with using Rust.
+- My company has not seriously considered Rust for any use.
+- I am unsure whether my company has considered using or currently uses Rust.
+- I don't work for a company or my company does not develop software of any kind.
+
+> **justification**
+>
+> We want to establish how reliant companies are on Rust.
+
+### In what technology domains is Rust used at your company?
+
+Select as many as apply:
+- backend application 
+- desktop application 
+- mobile application 
+- web application 
+- embedded application
+- other
+
+> **justification**
+>
+> We want to known roughly what technology stacks are being most often used.
+> 
+> **challenges**
+>
+> This can be ambiguous and hard to answer. For example, if you're building an operating 
+> system for a mobile phone, is that embedded, mobile, or something else.
+> We want to understand the "shape" of Rust usage, and this question only gets at that 
+> in one particular way.
+
+### Do you or your company use Rust at work?
+
+Select one:
+- Yes, I work with Rust full time
+- Yes, I work with Rust part time
+- No, but my company uses Rust 
+- No
+- Not sure 
+- Not applicable 
+
+> **REMOVED**
+>
+> This question was removed from the survey, because it was not sufficient in gathering the information we wanted.
 
-How reliant on Rust is your company?
+### For programmers - what languages, other than Rust, do you use at work?
+
+Free form:
 
-- It's the companies main language tech
-- It's a large part
-- It's a small part
-- We're experimenting
-- No use
+> **REMOVED**
+>
+> While this might be an interesting question, there's not too much that is directly actionable from it.
 
-And/or
+### Does your company plan to use Rust or evaluate Rust in the future?
 
-- Used in production across multiple projects
-- Used in production
-- Being actively developed
-- Experimented with
-- Being considered
+Select one:
+- Yes 
+- No
+- I don't know
 
-In what domains is Rust used at your company
+> **REMOVED**
+>
+> This data is better captured in other questions.
 
-- server networking, desktop application, mobile application, web application, embedded, etc.
+### How could we make Rust more appealing to your company?
+
+Free form:
+
+> **REMOVED**
+>
+> This question is too vague and unlikely to yield any interesting answers.
+
+### How may developers at your company use Rust at work?
+
+- 1
+- 2-5
+- 6-10
+- 11-25
+- 25-100
+- More than 100
+- I don't know
+
+> **REMOVED**
+>
+> This question fails to consider companies with different sizes. One company with 100,000 developers
+> where more than 100 use Rust is different than a company with 110 developers. We can gauge the importance
+> of Rust to companies through more direct questions
+
+### Approximately how many total developers does your company employ?
+
+- under 10
+- 11-49
+- 50-99
+- 100-500
+- 100-500
+- 500-1,000
+- 1,000-10,000
+- 10,000-100,000
+- Over 100,000
```

Comment:
I would probably go for "over 10,000" in place of the last two (given the number of companies with more than 10000 devs)